addon/services/key-manager.ts

```diff
@@ -0,0 +1,201 @@
+import Service from '@ember/service';
```

Comment:
Can you call out if anything was changed in this file?

Reply:
Only refactoring. No functional changes.
reading config from `ember-config-service` to clean up
and using typescript to clarify, and reuse external types rather than redefine